src/Service/Action.php

```diff
@@ -680,4 +713,46 @@ public function getOption(string $optionName)
             return null;
         }
     }
+
+    /**
+     * 設定JSON RPC 呼叫參數
+     *
+     * @param string|null $method
+     * @param array $arguments
+     * @return ActionInterface
+     */
+    public function setRpcQuery(string $method = null, array $arguments = []): ActionInterface
+    {
+        $rpcClient = ServiceList::getRpcClient();
+
+        // the rpc unique id
```

Comment:
I think we should use [UUID](https://zh.wikipedia.org/zh-tw/%E9%80%9A%E7%94%A8%E5%94%AF%E4%B8%80%E8%AF%86%E5%88%AB%E7%A0%81) in here.

Reply:
Does it mean random uuid?
This is because the [link](https://zh.wikipedia.org/zh-tw/%E9%80%9A%E7%94%A8%E5%94%AF%E4%B8%80%E8%AF%86%E5%88%AB%E7%A0%81) contains five versions of UUID generation.
Should we use version 4?

Reply:
done.

Reply:
good